eskipfile/file.go

```diff
@@ -45,9 +45,16 @@ func Open(path string) (*Client, error) {
 	return &Client{routes}, nil
 }
 
+func (c Client) LoadAndParseAll() (routeInfos []*eskip.RouteInfo, err error) {
```

Comment:
well, it says usually uncountable, and I don't know if this is an unusual case. And any other more detailed explanation says that the plural form is wrong. And just asked a native, and she says it's wrong. Are you sure that it's good here?